Projects/MoveFlow/Sources/View/MovingFlowHousingType.swift

```diff
@@ -74,21 +81,24 @@ public struct MovingFlowHousingTypeView: View {
                         dismissButton: nil
                     )
                 )
-                VStack {
-                    Spacer()
-                    hButton.LargeButton(type: .ghost) {
-                        router.dismiss()
-                    } content: {
-                        hText(L10n.generalCancelButton)
-                    }
-                }
+                .hDisableScroll
+        }
+        .onAppear {
```

Comment:
Can we handle this from the view model somehow?
Now we reset addressInputModel every time when this views appear